test/controllers/who.is.making.filing.controller.spec.ts

```diff
@@ -15,7 +15,7 @@ import {
   PAGE_TITLE_ERROR,
   RADIO_BUTTON_AGENT_SELECTED,
   RADIO_BUTTON_SOMEONE_ELSE_SELECTED,
-  SERVICE_UNAVAILABLE,
+  SERVICE_UNAVAILABLE, UK_REGULATED_AGENT,
```

Comment:
minor change, i'd have the new import on a separate line like the others

Reply:
Updated to go on newline.